pallets/schemas/src/migration/v3.rs

```diff
@@ -0,0 +1,137 @@
+#[cfg(feature = "try-runtime")]
+use crate::types::SCHEMA_STORAGE_VERSION;
+use crate::{
+	pallet::{SchemaInfos, SchemaNameToIds},
+	Config, Pallet, SchemaId, SchemaName, LOG_TARGET,
+};
+use frame_support::{pallet_prelude::*, traits::OnRuntimeUpgrade, weights::Weight};
+use log;
+use sp_runtime::Saturating;
+#[cfg(feature = "try-runtime")]
+use sp_runtime::TryRuntimeError;
+use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
+
+/// get known schema names for mainnet
+#[cfg(feature = "frequency")]
+pub fn get_known_schemas() -> BTreeMap<SchemaId, Vec<u8>> {
+	BTreeMap::from([
+		(1, b"dsnp.tombstone".to_vec()),
+		(2, b"dsnp.broadcast".to_vec()),
+		(3, b"dsnp.reply".to_vec()),
+		(4, b"dsnp.reaction".to_vec()),
+		(5, b"dsnp.update".to_vec()),
+		(6, b"dsnp.profile".to_vec()),
+		(7, b"dsnp.public-key".to_vec()),
+		(8, b"dsnp.public-follows".to_vec()),
+		(9, b"dsnp.private-follows".to_vec()),
+		(10, b"dsnp.private-connections".to_vec()),
+	])
+}
+/// get known schema names for rococo
+#[cfg(not(feature = "frequency"))]
```

Comment:
While this is good for testing, this breaks local upgrades. We need to either limit it to Rococo, skip it, or don't throw errors if the schema is not found.

Reply:
I've modified this logic but we don't through any errors if the schema id is not found.